src/olympia/discovery/tests/test_admin.py

```diff
@@ -290,7 +290,7 @@ def test_can_add_primary_hero_with_discovery_edit_permission(self):
                 'custom_addon_name': 'Xäxâxàxaxaxa !',
                 'recommendable': True,
                 'primaryhero-0-gradient_color': '#054096',
-                'primaryhero-0-image': 'Ubo@2x.jpg',
+                'primaryhero-0-select_image': 'Ubo@2x.jpg',
```

Comment:
I'm surprised this actually works - is that what the form actually sends in the browser?  I would have thought the POST data (use devtools in browser) would be sending an int representing the `PrimaryHeroImage.id`

Reply:
You're right - it passes an int!